### PR TITLE
(BIDS-2212) rename epoch to block fork variables

### PR DESCRIPTION
**Pull request by MauserBitfly**
### <samp>🤖 Generated by Copilot at d95c789</samp>

This pull request fixes a typo in the Eth1 chain configuration parameters and adds a new parameter for the Constantinople fork block number. It also updates the code that calculates the Eth1 block reward to use the correct fork block numbers.

### CODE REVIEW

File: config/mainnet.chain.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ CAPELLA_FORK_EPOCH: 194048 # April 12, 2023, 10:27:35pm UTC
 DENEB_FORK_VERSION: 0x04000000
 DENEB_FORK_EPOCH: 18446744073709551615
 # Byzantium
-BYZANTIUM_FORK_EPOCH: 4370000
+BYZANTIUM_FORK_BLOCK: 4370000
 # Constantinople
 CONSTANTINOPLE_FORK_EPOCH: 7280000
```
Comment (D13ce) [inline config/mainnet.chain.yml:53]:
```suggestion
CONSTANTINOPLE_FORK_BLOCK: 7280000
```

File: config/prater.chain.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ CAPELLA_FORK_EPOCH: 162304
 DENEB_FORK_VERSION: 0x04001020
 DENEB_FORK_EPOCH: 18446744073709551615
 # Byzantium
-BYZANTIUM_FORK_EPOCH: 0
+BYZANTIUM_FORK_BLOCK: 0
 # Constantinople
 CONSTANTINOPLE_FORK_EPOCH: 0
```
Comment (D13ce) [inline config/prater.chain.yml:47]:
```suggestion
CONSTANTINOPLE_FORK_BLOCK: 0
```

**Review by D13ce — CHANGES_REQUESTED** [review]

File: config/mainnet.chain.yml (hunk 1 above)
Comment (MauserBitfly) [reply config/mainnet.chain.yml]: renamed

File: config/prater.chain.yml (hunk 2 above)
Comment (MauserBitfly) [reply config/prater.chain.yml]: renamed

**Pull request merged**